modules/account-lib/src/coin/eth/iface.ts

```diff
@@ -1,4 +1,3 @@
-import BigNumber from 'bignumber.js';

```

Comment:
?

Reply:
This was an unused import. We had to fix it because of the eslint update